util/Makefile

```
@@ -4,17 +4,24 @@ TARGET = zenfs
 
 CC ?= gcc
 CXX ?= g++
+OBJCOPY ?= objcopy
 
 CXXFLAGS = $(shell pkg-config --cflags rocksdb)
 LIBS = $(shell pkg-config --static --libs rocksdb)
 
 CXXFLAGS +=  $(EXTRA_CXXFLAGS)
 LDFLAGS +=  $(EXTRA_LDFLAGS)
 
-all: $(TARGET)
+all: $(TARGET) generate_dbg
+
+generate_dbg:
+	@$(OBJCOPY) --only-keep-debug $(TARGET) $(TARGET).dbg
+	@$(OBJCOPY) --strip-debug $(TARGET)
+
+debug: $(TARGET)
```

```suggestion
debug: $(TARGET) CXXFLAGS+="-Og -g"
```
I don't know if the append works as a target specific variable, but I think it would be good to use these options for debug builds.

---

Added -g to main target, without it, gdb was not able to load symbols from the .dbg file.